content/v2.0/introduction/getting-started.md

```diff
@@ -0,0 +1,1382 @@
+---
+title: "Getting Started"
+order: 10
+aliases:
+  - "/getting-started"
+  - "/introduction/getting-started"
+---
+
+<p id="getting-started-lead" class="lead">
+  Hi there! <br/><br/>
+  You're reading the 'Getting Started' guide for Hanami 2.0.
+  It's still in <strong>alpha</strong>, so this guide is a little rough.
+  An obvious example is that the generator
+  (<code>bundle exec hanami generate ...</code>) commands listed below
+  are not implemented yet.<br/><br/>
+  Likewise, we'll instruct you to clone a template repository.
+  We will have a `hanami new` command that will create all these files
+  but that's not done yet either.<br/><br/>
+  It'll have the options you expect for customizing your project:
+  specifying a database engine, specifying a view template engine, etc.
+  Please note that our application template uses `slim` for view templating
+  but Hanami 2 will support <a href="https://github.com/rtomayko/tilt">any templating engine</a>.<br/><br/>
+  The 'Getting Started' guide generally wants you to follow along,
+  but it might be better to just read through it at this point.
+
+  <!-- TODO: I liked the 'letter' that was here but we should write a whole new one for Hanami 2.0 -->
+</p>
+
+<br>
+<hr>
+
+## Prerequisites
+
+Before we get started, let's get some prerequisites out of the way.
+First, we're going to assume basic knowledge of developing web applications.
+
+You should also be familiar with
+[Bundler](http://bundler.io),
+[Rake](http://rake.rubyforge.org),
+working with a terminal,
+and building apps using the
+[Model, View, Controller](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller) paradigm.
+
+Lastly, in this guide, we'll be using an [SQLite](https://sqlite.org/) database.
+If you want to follow along, make sure you have a working installation of Ruby 2.6+ and SQLite 3+ on your system.
+
+
+## Create a New Hanami Project
+
+<p class="alpha">
+  The `hanami new` command in this section doesn't exist for Hanami 2 alpha yet.<br/>
+  Instead run:<br/>
+  <code>
```

Comment:
In addition to that we could use `<code>hanami new</code>` instead of normal single backtick command highlighting in those alpha paragraphs and change the CSS rules from `p.alpha code` to `p.alpha .language-shell` etc so we have the usual command highlighting as well.

In combination this would look like:

![alpha-paragraph-highlighting](https://user-images.githubusercontent.com/675705/166160386-c4852abf-10ca-439f-bd19-1354ceb1fde8.png)